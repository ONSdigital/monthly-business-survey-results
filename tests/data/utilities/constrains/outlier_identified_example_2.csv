reference,period,spp_form_id,questioncode,value,outlier_weight,winsorised_value
1,202001,14,42,400,0.9,360
1,202001,14,43,800,0.8,640
1,202001,14,40,1200,1,1200
