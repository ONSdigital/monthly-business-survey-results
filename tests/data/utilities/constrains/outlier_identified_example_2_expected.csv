reference,period,spp_form_id,questioncode,value,outlier_weight,winsorised_value,post_winsorised
1,202001,14,42,400,0.9,360,FALSE
1,202001,14,43,800,0.8,640,FALSE
1,202001,14,40,1200,0.833333333,1000,TRUE
